modules/tensor_mechanics/include/materials/lagrangian/ComputeSimoHughesJ2PlasticityStress.h

```diff
@@ -0,0 +1,71 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ComputeLagrangianStressPK1.h"
+#include "GuaranteeConsumer.h"
+#include "ElasticityTensorTools.h"
```

Comment:
I may find out below, but do you need this include?

Reply:
Yes.

Reply:
This is for extracting the isotropic parameters.